.gitignore

```diff
@@ -133,5 +134,7 @@ dmypy.json
 batch/data/
 .DS_Store
 
-
 .venv/
+
+fly.toml
+
```

Comment:
Toptal gitignore file template is pretty much the standard and most future proof .gitignore file for django, I would recommend it over this particular one and hope it get changed. A copy of it is pasted down below:


Edit at https://www.toptal.com/developers/gitignore?templates=django